rllib/evaluation/tests/test_envs_that_crash.py

```diff
@@ -99,7 +100,7 @@ def test_crash_only_one_worker_during_sampling_but_ignore(self):
     def test_crash_only_one_worker_during_sampling_but_recreate(self):
         """Expect some sub-envs to fail (and not recover), but re-create worker."""
         config = (
-            PPOConfig()
+            PGConfig()
```

Comment:
Changed this back to PG to prove that it's just the change from PG to PPO that caused the timeouts.